src/runCIT.command

```diff
@@ -5,9 +5,9 @@ buildFolderPath=$ABSPATH/Build
 testLogPath=$buildFolderPath/CITLog.txt
 
 echo "******* Build and run CIT *******" 2>&1 | tee -a "$testLogPath"
-cd "$ABSPATH/WindowsAzureMessaging/WindowsAzureMessagingTest"
+cd "$ABSPATH/WindowsAzureMessaging"
 
-xcodebuild -scheme WindowsAzureMessagingTest -destination 'platform=iOS Simulator,name=iPhone SE,OS=11.4' test 2>&1 | tee -a "$testLogPath"
+xcodebuild -scheme WindowsAzureMessagingStatic -destination 'platform=iOS Simulator,name=iPhone 8' test 2>&1 | tee -a "$testLogPath"
```

Comment:
Why are we using an iPhone 8 and should we set a specific version of iOS?

Reply:
I just updated existing device on one, that should be available on all current SDKs.
About version, it's better to use without version, it's able to run tests on current simulators that we have in the system and do not force ourself on specific version.

Reply:
But maybe these settings required additional investigation